### PR TITLE
Updated Task Library Text Tasks Documentation

### PR DESCRIPTION
**Pull request by priankakariat**

### CODE REVIEW

File: tensorflow/lite/g3doc/inference_with_metadata/task_library/bert_nl_classifier.md [hunk 1]
```diff
@@ -120,6 +120,38 @@ See the
...
+
+```python
+# Imports
+from tflite_support.task import text
+from tflite_support.task import core
+from tflite_support.task import processor
+
+# Initialization
+base_options = core.BaseOptions(file_name=model_path)
+options = text.BertNLClassifierOptions(base_options=base_options)
```
Comment (khanhlvg) [inline tensorflow/lite/g3doc/inference_with_metadata/task_library/bert_nl_classifier.md:141]: As the default initializer is the only initializer supported all three NLP tasks, let's show only it.

```
classifier = text.BertNLClassifier.create_from_file(model_path)
```

**Review by khanhlvg — CHANGES_REQUESTED** [review]

File: tensorflow/lite/g3doc/inference_with_metadata/task_library/bert_nl_classifier.md (hunk 1 above)
Comment (priankakariatyml) [reply tensorflow/lite/g3doc/inference_with_metadata/task_library/bert_nl_classifier.md:141]: @kinaryml

Comment (priankakariatyml) [reply tensorflow/lite/g3doc/inference_with_metadata/task_library/bert_nl_classifier.md:141]: fixed

**Comment by gbaned** [comment]
Hi @khanhlvg Can you please review this PR ? Thank you!

**Review by khanhlvg — APPROVED** [review]

**Pull request merged**